### ISSUE TITLE
Updated to 4.6.3 --> Site is no longer loading correctly

### ISSUE THREAD

**Issue by xxtheatrainxx** [snipe/snipe-it#6054]
#### Please confirm you have done the following before posting your bug report:

- [ ] I have enabled debug mode 
- [ ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
I updated to 4.6.3 this morning and now the site isn't loading correctly.  I did mess up and update my composer, but I fixed that by copying the composer lock/json files from master and then running through the install another time.

**To Reproduce**
Honestly not sure 

**Expected behavior**
I'm not sure if the layout or design has changed since ~4.1 but the expectation is for the newer version to look something similar to that.

**Screenshots**
![image](https://user-images.githubusercontent.com/15128846/43793579-99c17ff0-9a41-11e8-9885-7e0e3a790ada.png)

**Server (please complete the following information):**
 - Snipe-IT Version:  4.6.3 
 - OS:  Win2k8
 - Web Server:  IIS
 - PHP Version:  5.1?

**Desktop (please complete the following information):**
 - OS:  Win10
 - Browser:  Chrome & IE
 - Version:  68 & 11.492

**Additional context**
- Is this a fresh install or an upgrade?  Both...converted to a git installation this morning but copied over 
       the files as suggested
- What method you used to install Snipe-IT:  Git

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

**Comment by snipe** [comment]
Looks like the CSS/JS isn't loading. What's in your browser's error console?

**Comment by xxtheatrainxx** [comment]
Learn something new every day, I do.  I'm not sure what all of this means, though.  Do I need to pull the files listed below and replace them again or do I need to install something on the server (or something else entirely)?

`Refused to apply style from 'http://server-name/snipeit/index.php/css/dist/all.css?id=98db4e9b7650453c8b00' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
all.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)
index.php:1 Refused to execute script from 'http://server-name/snipeit/index.php/js/dist/all.js?id=9d02373ef452329336d3' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
bootstrap-table.min.js:8 Uncaught ReferenceError: jQuery is not defined
    at bootstrap-table.min.js:8
bootstrap-table-mobile.js:131 Uncaught ReferenceError: jQuery is not defined
    at bootstrap-table-mobile.js:131
bootstrap-table-export.js?v=1:105 Uncaught ReferenceError: jQuery is not defined
    at bootstrap-table-export.js?v=1:105
jquery.base64.js:1 Uncaught ReferenceError: jQuery is not defined
    at jquery.base64.js:1
tableExport.min.js:77 Uncaught ReferenceError: jQuery is not defined
    at tableExport.min.js:77
bootstrap-table-cookie.js?v=1:413 Uncaught ReferenceError: jQuery is not defined
    at bootstrap-table-cookie.js?v=1:413
index.php:942 Uncaught ReferenceError: $ is not defined
    at index.php:942
index.php:1547 Uncaught ReferenceError: $ is not defined
    at index.php:1547
index.php:1581 Uncaught ReferenceError: $ is not defined
    at index.php:1581
index.php:1616 Uncaught ReferenceError: $ is not defined
    at index.php:1616
index.php:1 Refused to apply style from 'http://server-name/snipeit/index.php/css/dist/all.css?id=98db4e9b7650453c8b00' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.`

**Comment by xxtheatrainxx** [comment]
> Looks like the CSS/JS isn't loading.

How can I get these to load?  I have another (older) instance of Snipe IT running on this server and it hasn't been affected by this upgrade.  Do I need to start over completely or is there an "ah-ha" that I'm missing?

**Comment by xxtheatrainxx** [comment]
It seems like this is a common issue with IIS users.  I managed to get my "new" instance up and running by updating the URL rewrite rules and by turning on the rewrite engine.

Current (and previous) APP_URL in .env
![image](https://user-images.githubusercontent.com/15128846/44219523-dbf4cd80-a141-11e8-8661-55d080accd8b.png)

The RewriteEngine was **off** so I turned it **on**.  Also added in the RewriteBase (root is /assets) but that rule isn't supported by IIS according to the URL Rewrite import function
![image](https://user-images.githubusercontent.com/15128846/44219777-7a812e80-a142-11e8-944a-ecbaf2fc5cf3.png)

Imported the URL rewrite rules (1-1,2-1,3) from the new .htaccess file.  Had to disable 1-1.
![image](https://user-images.githubusercontent.com/15128846/44221437-11e88080-a147-11e8-8ccb-cfc9be8ab8c7.png)

This is the first time that I haven't had to include **/index.php/** in the URL since I've started using Snipe-IT.

I'm very green when it comes to web app admin so I'm not sure if these are "duh" kinda things or if they're specific to IIS.  Either way, thought it was worth mentioning for others experiencing similar issues.

-  had to clear the cache a couple of times when making changes to the .htaccess and .env file

-  **upgrade to git** instructions don't mention modifying or bringing over the old .htaccess file or needing to import new URL rewrite rules but I think doing this resolved my issue
